client/extensions/woocommerce/style.scss

```diff
@@ -26,6 +26,20 @@
 		border: 0;
 	}
 
+	.products__header {
```

Comment:
This is general comment and not on this PR.  Will this css file not get massive?  Is there a mechanism to break this up into the components?